{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:4.0",
                "name": "python3-lxml",
                "purl": "pkg:rpm/vmware/python3-lxml?distro=photon-4"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "4.6.3-1.ph4"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2021-4.0-0072",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2021-03-12T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-4.0-4"
        }
    ],
    "related": [
        "CVE-2021-28957",
        "CVE-2020-27783"
    ]
}